README.md

```diff
@@ -1 +1,121 @@
 # Service Binding Operator: Connect Applications with Services
+
+## Operator Goal - Binding Imported Applications with Operator-Backed Services
+
+The goal of the Service Binding Operator is to enable application authors to import an application and run it on OpenShift with backing services such as as databases, without having to perform manual configuration of secrets, configmaps, etc. 
+
+In order for the Service Binding Operator to bind an application to a backing service, the backing service operator must specify the information required to for the application to bind to the operator's service. The information must be specified in the operator's OLM descriptor from which it will be extracted to bind the application to the operator. The information can be specified in a secret, or in the "status" and/or "spec" section of the OLM.
+
+In order to make an imported application (for example, as a NodeJS application) connect to a backing services (for example as a database):
+
+* The app author (developer) creates a ServiceBindingRequest and specifies a label in the form of "connects-to." For example, to bind to an operator-backed database service created using a CR named "my-db", the label will be "connects-to=my-db".
+
+* The Service Binding Operator then:
+  * Reads backing service operator OLM descriptor to discover the binding attributes
+  * Creates a binding secret for the database
+  * Injects environment variables into the applications's DeploymentConfig, Deployment, Replicaset
+
+Here is an example of the "bind-able" operator OLM Descriptor - in ths case for a PostgreSQL database backing operator:
+
+```
```

Comment:
After the triple backtick you can add a ` yaml` (space followed by the format), therefore GitHub will render colorized syntax. For instance:
```
yaml:
  entry: value
  int: 1
```
Versus:
``` yaml
yaml:
  entry: value
  int: 1
```